specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimloggers.json

```diff
@@ -344,13 +353,6 @@
           {
             "$ref": "./apimanagement.json#/parameters/IfMatchRequiredParameter"
           },
-          {
-            "name": "force",
```

Comment:
>name [](start = 13, length = 4)

Another potential breaking change?

Reply:
@majastrz This is a new api-version and not published yet.